lnrpc/walletrpc/walletkit_server.go

```diff
@@ -744,13 +749,12 @@ func (w *WalletKit) PendingSweeps(ctx context.Context,
 	// Convert them into their respective RPC format.
 	rpcPendingSweeps := make([]*PendingSweep, 0, len(pendingInputs))
 	for _, pendingInput := range pendingInputs {
-		var witnessType WitnessType
-		if wt, ok := allWitnessTypes[pendingInput.WitnessType]; ok {
-			witnessType = wt
-		} else {
-			log.Warnf("Unhandled witness type %v for input %v",
-				pendingInput.WitnessType, pendingInput.OutPoint)
-			witnessType = WitnessType_UNKNOWN_WITNESS
+		witnessType, ok := allWitnessTypes[pendingInput.WitnessType]
+		if !ok {
+			return nil, fmt.Errorf(
```

Comment:
nit: logging and error function calls get an exception in the formatting to help distinguish them from actual ("business logic") function calls: https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#exception-for-log-and-error-message-formatting.

Reply:
I see, makes sense. I've changed it from

```
return nil fmt.Errorf(
        "an error",
)
```
to

```
return nil fmt.Errorf(
        "an error")
```

Reply:
Actually, it should even be:
```
return nil, fmt.Errorf( "an error")
```

So in this case:
```
			return nil, fmt.Errorf("unhandled witness type %v for "+
				"input %v", pendingInput.WitnessType,
				pendingInput.OutPoint)
```

Reply:
Right. Fixed it.